### PR TITLE
[CDAP-16202] Add e2e tests for pipeline actions

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Adds tests for zoom-in/out, fit-to-screen & Align pipeline actions 
- Adds a minor data-cy attribute to Feature Heading. This is to avoid tool tips for pipeline action buttons while doing screenshot diffs.

Build: https://builds.cask.co/browse/CDAP-UDUT484
E2E Test: https://builds.cask.co/browse/IT-UPD2100-1

### CODE REVIEW

File: cdap-ui/cypress/integration/pipeline.canvas.actions.spec.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      expect(stages.length).eq(3);
+    });
+  });
+
+  it('Should fit pipeline to screen', () => {
+    cy.visit('/pipelines/ns/default/studio');
+    cy.create_complex_pipeline();
+    cy.get('[data-cy="plugin-node-BigQueryMultiTable-batchsink-7"]').should('be.visible');
+    cy.get('[data-cy="plugin-node-BigQueryMultiTable-batchsink-6"]').should('be.visible');
+    cy.get('[data-cy="pipeline-zoom-in-control"]').click();
```
Comment (yukiej) [inline cdap-ui/cypress/integration/pipeline.canvas.actions.spec.ts]: It really helped readability in the previous test to set the selector string to a variable - could do for the other control buttons too.

File: cdap-ui/cypress/integration/pipeline.canvas.actions.spec.ts [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        };
+      });
+    });
+  });
+
+  beforeEach(() => {
+    getArtifactsPoll(headers);
+  });
+
+  it.skip('Should correctly undo/redo actions done by the user', () => {
```
Comment (yukiej) [inline cdap-ui/cypress/integration/pipeline.canvas.actions.spec.ts]: why is this one skipped?

Comment (ajainarayanan) [reply cdap-ui/cypress/integration/pipeline.canvas.actions.spec.ts]: 🤦‍♂ development changes. Will revert it back.

**Review by yukiej — APPROVED** [review]
lgtm!

**Pull request merged**